### PR TITLE
As User - I want to be offered better options after creating a service

### PR DESCRIPTION
**Pull request by gitstart**
Fixes https://github.com/amplication/amplication/issues/6104
### Demo video while creating service 
https://jam.dev/c/01995d98-c622-4b5c-8e48-01def8032257

### Demo video while onboarding 
https://jam.dev/c/0babac7d-7c4e-4541-bc15-c7a40caeb1d2

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx [hunk 1]
```diff
@@ -72,14 +84,17 @@ export const CreateServiceNextSteps: React.FC<WizardStepProps> = ({
...
+          <CircleBadge
+            color={isOnBoarding ? "#8DD9B9" : "#f85b6e"}
+            size="medium"
+          >
+            <Icon icon={isOnBoarding ? "users" : "plugins"} size="small" />
           </CircleBadge>
           <div className={`${className}__link_box__description`}>
-            <div>Create</div>
-            <div>another service</div>
+            <div>{isOnBoarding ? "Invite " : "Add plugins"}</div>
```
Comment (barshimi) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx:96]: use the object

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx [hunk 2]
```diff
@@ -72,14 +84,17 @@ export const CreateServiceNextSteps: React.FC<WizardStepProps> = ({
...
+            <div>{isOnBoarding ? "my team" : "to my service"}</div>
```
Comment (barshimi) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx:97]: also here

File: packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx [hunk 3]
```diff
@@ -243,6 +243,17 @@ const CreateServiceWizard: React.FC<Props> = ({
       ? CREATE_SERVICE_PATTERN
       : ONBOARDING_PATTERN;
 
+  const serviceNextStepStatus = {
```
Comment (barshimi) [inline packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx:246]: assign once use multiple:
const isSignupUser = signupCookie === "1"
...
isSignupUser ? "users" : "plugins"
etc

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx [hunk 4]
```diff
@@ -29,13 +43,18 @@ export const CreateServiceNextSteps: React.FC<WizardStepProps> = ({
     );
   }, [currentWorkspace, currentProject, serviceResults?.resource]);
 
-  const handleClickCreateNewService = useCallback(() => {
+  const handleClickRoute = useCallback(() => {
```
Comment (barshimi) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx:46]: great !

**Comment by mulygottlieb** [comment]
@barshimi can you please verify if all your comments are addressed and comment further / approve this?

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx [hunk 5]
```diff
@@ -29,12 +31,22 @@ export const CreateServiceNextSteps: React.FC<WizardStepProps> = ({
...
     trackWizardPageEvent(
       AnalyticsEventNames.ServiceWizardStep_Finish_CTAClicked,
-      { action: "Create Another Service" }
+      { action: "Add Plugins" }
     );
-    window.location.reload();
+    history.push(
+      `/${currentWorkspace.id}/${currentProject.id}/${serviceResults?.resource?.id}/plugins/catalog`
+    );
+  }, []);
```
[the remark this answers is not in the document]
Comment (morhag90) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx]: Please add the Ids and not all the objects: 

image

Thanks!

[the remark this answers is not in the document]
Comment (jaysoni-ash42) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx]: @morhag90  eventActionName is not an object it is a string according to user if it is signup user then
it is "Invite Team" or it is "Add Plugins"

[the remark this answers is not in the document]
Comment (morhag90) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceNextSteps.tsx]: The meaning is for the dependency of useCallback. You need to pass there: currentWorkspace.id and not currentWorkspace, etc...

**Review by morhag90 — APPROVED** [review]

**Pull request merged**